benchmarking/run_custom_dataset.sh

```diff
@@ -8,6 +8,7 @@ python src/evaluator.py \
 --num-workers 1 \
 --timeout 600 \
 --input-file-path "<CUSTOM DATASET PATH HERE>" \
+--save-llm-responses False \
 --llm-api "sambastudio"
```

Comment:
Could we add in `--sampling-params '{"max_tokens_to_generate": 256}'` here to let the user know they can control this param in this script?

Reply:
sampling params for max tokens has been added to bash process. I've also added the changes to make it work in streamlit. Thanks for the feedback!